apps/test-can/new-can-rx-test/main.cpp

```diff
@@ -0,0 +1,76 @@
+#include <vector>
+
+#include "CANBus.h"
+#include "CANMsg.h"
+
+static int received_msgs = 0;
+
+CANBus bus(CAN1_RX, CAN1_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+
+EventQueue queue;
+
```

Comment:
i dont think we should merge this into master. its a hot mess lol

Reply:
i was just trying to see if the new method worked and it does. if u rly want a test-app for the new can method, you'd have to clean this stuff up